barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -263,10 +264,13 @@ template <typename Curve> class ZeroMorphProver_ {
     }
 
     /**
-     * @brief Compute combined evaluation and degree-check quotient polynomial pi
-     * @details Compute univariate quotient pi, where
+     * @brief Compute combined evaluation and degree-check polynomial pi
+     * @details Compute univariate polynomial pi, where
+     *
+     *  pi = (\zeta_c + z*Z_x) X^{N_{max}-(N-1)}
```

Comment:
any reason you removed some of the details from this comment? I think it can't hurt to have

Reply:
oh, just because the quotient is going to be computed in KZG and it might not even be a quotient (i.e. in IPA it's not )

Reply:
added what will happen if we instantiate zeromorph with kzg so we can keep the details